app/scripts/directives/overview/mobileClientRow.js

```diff
@@ -67,6 +67,26 @@
     row.updateServicesInfo = function() {
       if (row.services) {
         row.servicesNotBoundCount = row.services.length - row.bindings.length;
+        var boundServices = 'Bound Services';
```

Comment:
Can we move the donut configuration/setup to `$onInit`? I think only `row.data` needs to be updated every change.